tests/watcher/utils/watcher.rs

```diff
@@ -74,50 +129,30 @@ impl WatcherTest {
         let post_id = post.create_id();
         let post_json = to_vec(post)?;
         let url = format!("pubky://{}/pub/pubky.app/posts/{}", user_id, post_id);
-        self.client.put(url.as_str(), &post_json).await?;
+        // Write the post in the pubky.app repository
+        PubkyConnector::get_pubky_client()?
```

Comment:
We could create a single `let pubky_client = PubkyConnector::get_pubky_client()?; ` instead of performing it 3 times